components/embl-logo/embl-logo.config.yml

```diff
@@ -16,6 +16,7 @@ context:
   component-type: embl-element
   href: http://www.embl.de
```

Comment:
There's also an `href` in addition to `logo_href`, I'm guessing this accidental?

Reply:
this is 'technically' ok - as the `href` is the `.yml` not actually in the code as `href="#"`.

But this does bring up the lack of consistency with the key naming for this across components.

I think something like `component_link` or `component_href` would be better.

I'll refactor so they're all on the same hymn sheet.

Reply:
What I was getting at is you've added a `logo_href`, I'm guessing you just forgot to delete the old one? Or is it used somewhere I'm not seeing?

Reply:
hmmm, now why wasn't that used before :sob:

I'll still go through and standardise the naming convention and do another push